src/device/gpoddevice.cpp

```diff
@@ -182,6 +182,19 @@ bool GPodDevice::CopyToStorage(const CopyJob &job) {
     return false;
   }
 
+  // Put the track in the playlist, if one is specified
+  if(!job.playlist_.isEmpty()) {
```

Comment:
Can you fix the code style here to 
if (!job.playlist_.isEmpty()) {
and the indent to 2 spaces